playwright.config.ts

```diff
@@ -47,7 +47,7 @@ const config: PlaywrightTestConfig = {
   // We need just a little more time on CI:
   timeout: process.env.CI ? 3 * 60_000 : 60_000,
   webServer: {
-    command: "npm run start",
+    command: "cd ./e2e/browser/testApp/ && npm run dev -- -p 3001",
     port: 3001,
```

Comment:
I have set it explicitly to 3001, because leaving this to default it will try 3000 and move to 3001 if 3000 isn't available, which caused the tests to run against a wrong app instance locally that was still running on 3000, so I'd prefer to be explicit and have the test crash if the port is unavailable.

Reply:
Okay, that's fair

Reply:
I'll let you imagine my confusion trying to figure out why the authn browser-based tests were spinning up the notifications app.

Reply:
yeah, totally!

Reply:
maybe we can make `npm run dev` actually set the port number?

Reply:
I think it's okay that `npm run dev` is a thin layer on top of `next dev`, and options are passed explicitly